src/Button/UploadButton/UploadButton.less

```diff
@@ -0,0 +1,21 @@
+.react-geo-uploadbutton{
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  overflow: hidden;
+
+  input[type="file"] {
+    cursor: pointer;
+    width: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+    padding: 0;
+    opacity: 0;
```

Comment:
This renders fine in all relevant browsers?

Reply:
Checkt with Chrome and IE.